engine/table/build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
 
     Classpaths.inheritGroovy(project, 'groovy', 'implementation')
     Classpaths.inheritGroovy(project, 'groovy-json', 'implementation')
-    Classpaths.inheritJavaParser(project)
+    Classpaths.inheritJavaParser(project, 'compile')
```

Comment:
I am confused by this change (and the other ones in this commit). Shouldn't we be using `compileOnly` or `implementation`? I think you intend this commit to not change anything, and that you're correcting in subsequent commits, in which case this can just be resolved.

Reply:
Correct - this commit was meant to keep everything the same and it was fixed one-by-one later.